components/atom/videoPlayer/src/components/NativePlayer.js

```diff
@@ -3,13 +3,19 @@ import {useRef} from 'react'
 import PropTypes from 'prop-types'
 
 import useGetBlobAsVideoSrcEffect from '../hooks/useGetBlobAsVideoSrcEffect.js'
+import {BASE_CLASS} from '../settings.js'
 
 const NativePlayer = ({src}) => {
   const videoNode = useRef(null)
   const videoSrc = useGetBlobAsVideoSrcEffect({src, videoNode})
 
   return (
-    <video ref={videoNode} title="Native video player" controls>
+    <video
+      className={`${BASE_CLASS}-nativePlayer`}
+      ref={videoNode}
+      title="Native video player"
```

Comment:
I wonder if we could keep those `title` values (as in https://github.com/SUI-Components/sui-components/pull/2529/files#diff-f06711f0e52367b36a693d58e6b8fe7ac792d218252b72cb7beaaf28b66174c4R80 ) on some config file?